ringpop.go

```diff
@@ -366,6 +385,10 @@ func (rp *Ringpop) GetReachableMembers() []string {
 	return rp.node.GetReachableMembers()
 }
 
+func (rp *Ringpop) CountReachableMembers() int {
```

Comment:
Thinking about this makes total sense, but it does not play nice with `len(slice)` and `LookupN`. They give compilation errors if you don't cast from `uint` back to `int` for assigning and comparison. I will keep it `int` for now to keep the code more readable.